plugins/module_utils/foreman_helper.py

```diff
@@ -360,15 +380,28 @@ def _update_entity(self, resource, desired_entity, current_entity, params, entit
         desired_entity = _flatten_entity(desired_entity, entity_spec)
         current_entity = _flatten_entity(current_entity, entity_spec)
         for key, value in desired_entity.items():
-            if current_entity.get(key) == value:
-                continue
-            payload[key] = value
+            # query entity_spec[key] for special treatment
+            # spec = entity_spec.get(key)
+            # if spec:
+                # properties with 'check_type' ???
```

Comment:
```
plugins/module_utils/foreman_helper.py:386:17: E117 over-indented (comment)
plugins/module_utils/foreman_helper.py:387:17: E117 over-indented (comment)
plugins/module_utils/foreman_helper.py:388:17: E117 over-indented (comment)
plugins/module_utils/foreman_helper.py:389:17: E117 over-indented (comment)
```

:stuck_out_tongue:

Reply:
I guess we should not bother this, until we need it.